site/content/faq/500-what-about-typescript-support.md

```diff
@@ -10,3 +10,5 @@ To declare the type of a reactive variable in a Svelte template, you should use
 let x: number;
 $: x = count + 1;
 ```
+
+Also, to import an interface please use [TypeScript's `import type` syntax](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export).
```

Comment:
Small suggestion:

```suggestion
To `import` a type or interface make sure to use [TypeScript's `type` modifier](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export):
```

I would also add a tiny example (I'm not sure how to add it to the GitHub's code block)

```diff
- import { SomeInterface } from './SomeFile';
+ import type { SomeInterface } from './SomeFile';
```

Reply:
I added the example as well